### PR TITLE
enhance documentation of caching, continuation of #914

### PR DESCRIPTION
**Pull request by SophieHerbst**
### Before merging …

- [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

File: docs/source/v1.9.md.inc [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 - We removed the `Execution` section from configuration options documentation and
   replaced it with new, more explicit sections (namely, Caching, Parallelization,
-  Logging, and Error handling). (#914 by @hoechenberger)
+  Logging, and Error handling), and enhanced documentation. (#914 by @hoechenberger, #916 by @SophieHerbst)
```
Comment (hoechenberger) [inline docs/source/v1.9.md.inc:23]:
```suggestion
  Logging, and Error handling), and enhanced documentation. (#914, #916 by @hoechenberger and @SophieHerbst)
```

**Comment by hoechenberger** [comment]
While we're talking caching … I would love to have all those parameters named `cache_*` and not `memory_*`.

Thoughts?

@larsoner

**Comment by larsoner** [comment]
Just adding a cross-ref to #916 since once we document the command-line options we should ideally cross-link the `--no-cache` in this PR to it

**Comment by larsoner** [comment]
Thanks @SophieHerbst !

**Pull request merged**